src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,308 @@
+    # Load Conditional Data
+    self.assertEqual(device_2['cbsdCategory'], 'B')
+    conditionals_device_2 = {
+        'cbsdCategory': device_2['cbsdCategory'],
+        'fccId': device_2['fccId'],
+        'cbsdSerialNumber': device_2['cbsdSerialNumber'],
+        'airInterface': device_2['airInterface'],
+        'installationParam': device_2['installationParam'],
+        'measCapability': device_2['measCapability']
+    }
+    self.assertEqual(device_4['cbsdCategory'], 'B')
+    conditionals_device_4 = {
+        'cbsdCategory': device_4['cbsdCategory'],
+        'fccId': device_4['fccId'],
+        'cbsdSerialNumber': device_4['cbsdSerialNumber'],
+        'airInterface': device_4['airInterface'],
+        'installationParam': device_4['installationParam'],
+        'measCapability': device_4['measCapability']
+    }
+    self.assertEqual(device_8['cbsdCategory'], 'B')
+    conditionals_device_8 = {
+        'cbsdCategory': device_8['cbsdCategory'],
+        'fccId': device_8['fccId'],
+        'cbsdSerialNumber': device_8['cbsdSerialNumber'],
+        'airInterface': device_8['airInterface'],
+        'installationParam': device_8['installationParam'],
+        'measCapability': device_8['measCapability']
+    }
+
+    conditionals = {'registrationData': [conditionals_device_2, conditionals_device_4, conditionals_device_8]}
+    # Remove conditionals from registration
+    del device_2['cbsdCategory']
+    del device_2['airInterface']
+    del device_2['installationParam']
+    del device_2['measCapability']
+    del device_4['cbsdCategory']
+    del device_4['airInterface']
+    del device_4['installationParam']
+    del device_4['measCapability']
+    del device_8['cbsdCategory']
+    del device_8['airInterface']
+    del device_8['installationParam']
+    del device_8['measCapability']
+
+    # Load GWPZ Record
+    gwpz_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+
+    # Load FSS record
+    fss_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+
+    # Load ESC record
+    esc_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+
+    # Load PPA record
+    ppa_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0
+
+    ppa_record_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                 [pal_record],
+                                                                 pal_low_frequency,
+                                                                 pal_high_frequency,
+                                                                 'test_user_1')
+    # Define DPAs
```

Comment:
the PPA is missing the CbsdReferenceId

Reply:
These are DPAs, are you referring to line 129 ? We will take a look of the PPA test data.